dist/angular-google-analytics.js

```diff
@@ -696,12 +696,25 @@
             if (angular.isDefined(noninteraction)) {
               opt_fieldObject.nonInteraction = !!noninteraction;
             }
+            
+            if (angular.isDefined(transport) && angular.isDefined(href)) {
+                opt_fieldObject.transport = transport;
+                opt_fieldObject.hitCallback = loadPage;
+            } else if (angular.isDefined(transport) || angular.isDefined(href)) {
+                alert("transport and outbound href required")
+            }
+            
             if (angular.isObject(custom)) {
               angular.extend(opt_fieldObject, custom);
             }
             if (!angular.isDefined(opt_fieldObject.page)) {
               opt_fieldObject.page = getUrl();
             }
+            
+            function loadPage() {
```

Comment:
This function should be declared before the reference to it, sensibly in the if-block on line 700 where it is solely used.